00 Intro/00 BoilerPlate/README_es.md

```diff
@@ -0,0 +1,152 @@
+
+
+````
+npm init
+````
+
+![npm init](../../99 Readme Resources/02 Webpack/Demo00_npminit.png "Demo00_npminit.png")
+
+- Instala **webpack** de forma local, como dependencia de desarrollo (la razón por la que es recomendado instalarlo de forma local en el proyecto y no global es porque será mas fácil ejecutarlo, por ejemplo, en una máquina limpia, sin obligar a que se instale de forma global ningun paquete como nodejs).
+
+````
+npm install webpack --save-dev
+````
+
+- Para lanzar webpack de forma local como hemos comentado en el punto anterior, modificaremos el archivo **package.json** y añadiremos la siguiente propiedad `"start": "webpack"` dentro del objeto scripts. Con este cambio, podremos lanzar webpack con el comando npm `npm start`.
+
+ Ahora, nuestro archivo **package.json** quedaría de la siguiente forma:
+
+![package.json](../../99 Readme Resources/02 Webpack/Demo00_webpackinstall.png "Demo00_webpackinstall.png")
+
+- Escibiremos código ES6, pero necesitamos traducirlo a ES5, para ello instalaremos `babel-core` mas `babe-preset-es2015` y lo guardaremos como dependencia de desarrollo en el archivo **package.json** que ha sido generado previamente.
+
+````
+npm install babel-core --save-dev
+npm install babel-preset-es2015 --save-dev
+````
+
+- Necesitamos instalar un "loader" (se explicará en los siguientes módulos) con el fin de que webpack sea capaz de usar `babel-core`.
+
+````
+npm install babel-loader --save-dev
+````
+
+Nuestro archivo **package.json** quedaría de la siguiente forma:
+
+![package.json](../../99 Readme Resources/02 Webpack/Demo00_packagejson.png "Demo00_packagejson.png")
+
+
+- Ahora creamos un archivo javascript **students.js** , que incluirá la sintaxis ES6.
+
+```javascript
+// Usaremos algunas características ES6
+const averageScore = "90";
+const messageToDisplay = `average score ${averageScore}`;
+
+document.write(messageToDisplay);
+```
+
+- Vamos a configurar ahora webpack, creamos un archivo vacío **webpack.config.js**, e indicamos que js tendremos como punto de partida.
+
+```javascript
+module.exports = {
+  entry: ["./students.js"],
+  output: {
+    filename: "bundle.js"
+  }
+}
+```
+
+- Ahora añadimos soporte para ES6, pediremos a webpack el manejo de todos los archivos js en el directorio del proyecto (excluyendo la subcarpeta `node_modules`) y traduciremos desde ES6 a ES5 (usando `babel-loader`).
+
+```javascript
+module.exports = {
+  entry: ["./students.js"],
+  output: {
+    filename: "bundle.js"
+  },
+
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: "babel-loader",
+        query: {
+          presets: ['es2015']
+        }
+      }
+    ]
+  }
+}
+```
+
+- Vamos a arrancar webpack desde la línea de comandos, escribiendo `npm start` y presionando enter.
+
+```
+npm start
```

Comment:
I don't understand what you do mean? I check the newest readme and it's equal as Readme_es.